File: Begginers/C++/tomadas_1930.cpp
#include <iostream>
#include <vector>
#include <sstream>

int main() {
    std::vector<int> reguasint;
    std::string input;
    getline(std::cin, input);
    std::istringstream iss(input);
    int regua;
    
    while (iss >> regua) {
          reguasint.push_back(regua);
          }

    int maximo = reguasint[0];

    for (int i = 0; i < reguasint.size() - 1; i++) {
        int conector = (maximo + reguasint[i + 1]) - 1;
        maximo = conector;
        }

        std::cout << maximo << std::endl;

        return 0;
}

File: Begginers/Python/tomadas_1930.py
reguas = input().split()
reguasint = [int(x) for x in reguas]
maximo = reguasint[0]

for i in range(len(reguasint) - 1):
    conector = (maximo + reguasint[i + 1]) - 1
    maximo = conector
            
print(maximo)
